content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
 
 [harvest]: https://www.getharvest.com/
 [accounts]: mailto:accounts@oddbird.net
+
+## Success Criteria:
```

Comment:
Not sure if the very end of the guide is the best place to put the section, open to suggestions

Reply:
I'd recommend after "Participation" and before "Finding Clients".

Reply:
After reading it again in context of the rest of the guide, I think it makes sense to have it at the end because it touches on several points introduced earlier, but I'm still open to moving it

Reply:
I think the end would be fine.